components/ui/animated-modal.tsx

```
@@ -2,242 +2,242 @@
+            </svg>
+        </button>
+    );
 };
 
 // Hook to detect clicks outside of a component.
 // Add it in a separate file, I've added here for simplicity
 export const useOutsideClick = (
-  ref: React.RefObject<HTMLDivElement>,
-  callback: Function
+    ref: React.RefObject<HTMLDivElement>,
+    callback: Function
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)